packages/scripts/internationalization/README.md

```diff
@@ -149,3 +150,50 @@ This step will download, unzip and transform files from XTM.
 | transform | Optional. Transformation to apply to file hierarchy. For now only `flatten` is accepted, putting all file directly under the target language folder. |
 
 
+### To Github
+This step will push downloaded i18n files to github.
+
+You need to pass github credentials as environment variables.
+
+```shell
+> GITHUB_LOGIN=XXX \
+  GITHUB_TOKEN=YYY \
+  talend-scripts i18n-extract
+```
+
+| Variable | Description |
+|---|---|
+| GITHUB_LOGIN | Github username that will be used to push to the locales repo |
+| GITHUB_TOKEN | Github token |
+
+
+`talend-i18n.json` configuration
+
+```json
+{
+  "github": {
+    "url": "https://github.com/Talend/locales.git"
+  }
+}
+```
+
+| Configuration | Description |
+|---|---|
+| url | The https git url. |
+
+This scripts will extract the version (major.minor) from files at project root.
+* lerna.json
+* package.json
+* pom.xml
+
+The i18n files will be pushed to a branch `release/{XTM_project}/{version}`, and a tag `{XTM_project}/{version}` is set at each new push.
```

Comment:
avoid `/` in tag name please prefer `-` and may be prefix with v to make it very different from branch name.
May be the branch can be different from release but just be `{XTM_project}/{version}`

Reply:
avoid `/` --> why ? We do that in Talend since a long time.
I added the `release` because it collide with tag name.

Reply:
This create folders.
Branches and tags are both reference. We should in that particular case keep all the references under the XTM_project.

`{XTM_project}/2.6.1` for tag
`{XTM_project}/2.x.x` for branch

Reply:
Done for branch name. Tag name stays `{XTM_project}/2.6`